app/views/mailing_list/steps/_already_qualified.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>We're sorry, but to sign up for our emails, you need to be interested in training to teach</h1>
+
+<p>
+  If you're already a qualified teacher from outside the UK, you can <%= link_to("find out more about teaching in England if you trained overseas", "/non-uk-teachers/teach-in-england-if-you-trained-overseas") %>.
+</p>
+
+<p>
+  If you're already qualified to teach in England, you can <%= link_to("find out more about how you can return to teaching", "/returning-to-teaching") %>.
```

Comment:
@EllieNodder Not sure if we need to tweak this a bit to say if you qualified **in England**

I think if I was a non-UK citizen who was qualified, I might read this sentence and think that this might apply to me, if I didn't know about QTS?

In which case, rather than say 'If you're already a qualified teacher from outside the UK...' should we say 'If you're already a qualified teacher with a qualification from outside the UK....' (very clunky!)

Reply:
have tweaked this slightly - does this work better?